indico/modules/auth/controllers.py

```diff
@@ -727,7 +727,11 @@ def _process_args(self):
 
     def _process(self):
         if 'token' in request.args:
-            data = secure_serializer.loads(request.args['token'], max_age=3600, salt='reset-password')
+            try:
+                data = secure_serializer.loads(request.args['token'], max_age=3600, salt='reset-password')
+            except (BadSignature, ValueError):
```

Comment:
When do you get a plain `ValueError`?

Reply:
E.g.:
https://indico.un.org/reset-password/?token=http://www.bestbuy.com